WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -445,47 +452,47 @@ public extension ArticleAsLivingDocViewModel.Event.Small {
     
     func eventDescriptionForTraitCollection(_ traitCollection: UITraitCollection, theme: Theme) -> NSAttributedString {
         if let lastTraitCollection = lastTraitCollection,
+           let lastTheme = lastTheme,
            let eventDescription = eventDescription {
-            if lastTraitCollection == traitCollection {
+            if lastTraitCollection == traitCollection && lastTheme == theme {
                 return eventDescription
             }
         }
         
         let font = UIFont.wmf_font(.italicSubheadline, compatibleWithTraitCollection: traitCollection)
-        let attributes = [NSAttributedString.Key.font: font,
-                          NSAttributedString.Key.foregroundColor: theme.colors.primaryText]
+        let linkAttributes = [NSAttributedString.Key.font: font,
+                          NSAttributedString.Key.foregroundColor: theme.colors.link]
+        let suffixAttributes = [NSAttributedString.Key.font: font,
+                          NSAttributedString.Key.foregroundColor: theme.colors.secondaryText]
         
-        let localizedString = String.localizedStringWithFormat(
+        let descriptionLocalizedString = String.localizedStringWithFormat(
             CommonStrings.smallChangeDescription,
             smallChanges.count)
+        let suffixString = CommonStrings.smallChangeDescriptionSuffix
         
-        let eventDescription = NSAttributedString(string: localizedString, attributes: attributes)
+        let linkAttString = NSAttributedString(string: descriptionLocalizedString, attributes: linkAttributes as [NSAttributedString.Key : Any])
+        let suffixAttrString = NSAttributedString(string: suffixString, attributes: suffixAttributes)
+        
+        let mutableAttString = NSMutableAttributedString()
+        mutableAttString.append(linkAttString)
+        mutableAttString.append(NSAttributedString(string: " "))
+        mutableAttString.append(suffixAttrString)
+        
+        guard let nsAttString = mutableAttString.copy() as? NSAttributedString else {
```

Comment:
Actually I purposefully like to copy mutable attributed strings & mutable nsstrings back over to immutable before assigning them anywhere so that they don't somehow get changed out from under me. It's sort of a safety pattern holdover for me from flagging certain Obj-C properties as `copy` (which was suggested for any class with a mutable counterpart). I am happy to rework if you know of a better way or if I'm thinking of it wrong but I do want to keep it in some form.